site/installation/migration.md

```diff
@@ -315,7 +347,7 @@ $ kpt version
 ```
 
 ```shell
-$ kpt pkg get $PKG_REPO
+$ kpt pkg get ${REPO_URI}/${PKG_PATH}@${VERSION}
```

Comment:
I feel like having 5 additional lines to make this parameterized is creating more noise than benefit. Also what if they don't use github? I think you should just assume they now how to use `pkg get`.

Reply:
Agree. Updated.